cli/cmd/main.go

```diff
@@ -185,6 +207,92 @@ func main() {
 					},
 				},
 			},
+			// TODO: Add help stuff.
+			{
+				Name:  "data",
+				Usage: "download synced data",
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     dataFlagDestination,
+						Required: true,
```

Comment:
Is there a convention to denote which one are required versus option? So when you type `viam data` optionals are in brackets or required are in angled brackets or something similar?

Reply:
Great suggestion! Updated to follow standard described [here](https://stackoverflow.com/questions/21503865/how-to-denote-that-a-command-line-argument-is-optional-when-printing-usage): <> for required, and [] for optional